packages/use-sync-external-store/package.json

```diff
@@ -2,6 +2,17 @@
   "name": "use-sync-external-store",
   "description": "Backwards compatible shim for React's useSyncExternalStore. Works with any React that supports hooks.",
   "version": "1.2.0",
+  "exports": {
+    ".": "./index.js",
+    "./with-selector": "./with-selector.js",
+    "./shim": {
+      "react-native": "./shim/index.native.js",
+      "default": "./shim/index.js"
+    },
+    "./shim/index.native": "./shim/index.native.js",
```

Comment:
For compat with bundlers that don't support the `react-native` condition. Though I don't know if we need it. I just copied it from  https://github.com/facebook/react/pull/24440

Reply:
https://github.com/facebook/react/blob/main/packages/use-sync-external-store/src/__tests__/useSyncExternalStoreNative-test.js#L45
It seems that this entry has been used in several places. I'm not sure if this is necessary.